.travis.yml

```diff
@@ -2,12 +2,14 @@ language:
   - python
 compiler:
   - gcc
-  - clang 
+  - clang
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -y libgmp-dev
 python:
   - "2.7"
+  - "3.5"
+  - "3.6"
```

Comment:
Any idea why Python 3.7 caused Travis tests to fail?    Weird if there's an incompatibility in that version and bummer if you go through this effort to add Python 3 and leave the lastest version behind  :)

Reply:
Due to this issue:
https://github.com/travis-ci/travis-ci/issues/9815

Travis CI doesn't support directly Python 3.7 only through matrix option in .travis.yml. I've added it.